conjur/argument_parser/_login_parser.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+from conjur.argument_parser.parser_utils import command_description, command_epilog, formatter, title_formatter
+
+
+class LoginParser:
+    def add_login_parser(self):
+        """
+        Method adds login parser functionality to parser
+        """
+        login_subparser = self._init_login_parser()
+        LoginParser._add_login_options(login_subparser)
+
+        return self
+
+    def _init_login_parser(self):
+        login_name = 'login - Log in to Conjur server'
+        login_usage = 'conjur [global options] login [options] [args]'
+
+        login_subparser = self.resource_subparsers.add_parser('login',
+                                                              help='Log in to Conjur server',
+                                                              description=command_description(login_name,
+                                                                                              login_usage),
+                                                              epilog=command_epilog('conjur login \t\t\t\t'
+                                                                                    'Prompts for the login name and password to log in to Conjur server\n'
+                                                                                    '    conjur login -i admin \t\t\t'
+                                                                                    'Prompts for password of the admin user to log in to Conjur server\n'
+                                                                                    '    conjur login -i admin -p Myp@SSw0rds!\t'
+                                                                                    'Logs the admin user in to Conjur server and saves the user and password '
+                                                                                    'in the local cache (netrc file)'),
```

Comment:
while we are here can you change this string to 'Logs the admin user in to Conjur server and saves the login ID and password in either the system's credential store or netrc'

this was something we should have changed previously ...

Reply:
@mbenita-Cyberark

Reply:
Done